aptos-move/framework/aptos-framework/sources/multisig_account.move

```diff
@@ -0,0 +1,1348 @@
+        transaction_id: u64,
+        transaction: MultisigTransaction,
+    }
+
+    /// Event emitted when an owner approves a transaction.
+    struct ApproveTransactionEvent has drop, store {
+        transaction_id: u64,
+        owner: address,
+        num_approvals: u64,
+    }
+
+    /// Event emitted when an owner rejects a transaction.
+    struct RejectTransactionEvent has drop, store {
+        transaction_id: u64,
+        owner: address,
+        num_rejections: u64,
+    }
+
+    /// Event emitted when a transaction is officially removed because the number of rejections has reached the
+    /// number of signatures required.
+    struct RemoveTransactionEvent has drop, store {
+        transaction_id: u64,
+        num_rejections: u64,
+        executor: address,
+    }
+
+    /// Event emitted when a transaction is executed.
+    struct TransactionExecutionSucceededEvent has drop, store {
+        executor: address,
+        transaction_id: u64,
+        transaction_payload: vector<u8>,
+        num_approvals: u64,
+    }
+
+    /// Event emitted when a transaction's execution failed.
+    struct TransactionExecutionFailedEvent has drop, store {
+        executor: address,
+        transaction_id: u64,
+        transaction_payload: vector<u8>,
+        num_approvals: u64,
+        execution_error: ExecutionError,
+    }
+
+    #[view]
+    public fun signatures_required(multisig_account: address): u64 acquires MultisigAccount {
+        borrow_global<MultisigAccount>(multisig_account).signatures_required
+    }
+
+    #[view]
+    public fun owners(multisig_account: address): vector<address> acquires MultisigAccount {
+        borrow_global<MultisigAccount>(multisig_account).owners
+    }
+
+    #[view]
+    public fun get_transaction(
+        multisig_account: address,
+        transaction_id: u64,
+    ): MultisigTransaction acquires MultisigAccount {
+        *table::borrow(&borrow_global<MultisigAccount>(multisig_account).transactions, transaction_id)
+    }
+
+    #[view]
+    public fun get_transaction_status(
+        multisig_account: address,
+        transaction_id: u64,
+    ): u64 acquires MultisigAccount {
+        let transaction = table::borrow(&borrow_global<MultisigAccount>(multisig_account).transactions, transaction_id);
+        transaction.status.status
+    }
+
+    #[view]
+    public fun get_next_transaction_payload(
+        multisig_account: address, provided_payload: vector<u8>): vector<u8> acquires MultisigAccount {
+        let multisig_account_resource = borrow_global<MultisigAccount>(multisig_account);
+        let transaction_id = multisig_account_resource.last_transaction_id + 1;
+        let transaction = table::borrow(&multisig_account_resource.transactions, transaction_id);
+
+        if (option::is_some(&transaction.payload)) {
+            *option::borrow(&transaction.payload)
+        } else {
+            provided_payload
+        }
+    }
+
+    #[view]
```

Comment:
Are view function callable now? Is there any doc update for this new tag?

Reply:
Yes there's a new API for view function now. We'll be adding docs for it soon.